### PR TITLE
[tests] Fix junit tests

### PR DESCRIPTION
**Pull request by lfasani**
Change-Id: I0a86101db980a7d562775f2ade01c5c9b0520f1b

### CODE REVIEW

File: plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/common/migration/DiagramMigrationTestCampaign09.java [hunk 1]
```diff
@@ -114,7 +114,13 @@ public void testEdgeLayout() throws Exception {
         openEditorOnDiagram("" + diagramID);
         assertTrue("", semanticModel instanceof TestCase);
         Representation representation = ((TestCase) semanticModel).getRepresentations().get(diagramID - 1);
-        checkEdgeLayout(representation);
+        if (diagramID == 1 || diagramID == 2 || diagramID == 3 || diagramID == 4 || diagramID == 7) {
+            // This particular diagram can has a difference of 1 pixel in the bendpoints position depending on the
```
Comment (lredor) [inline plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/common/migration/DiagramMigrationTestCampaign09.java:118]:
```suggestion
            // This particular diagram can has a difference of 2 pixels in the bendpoints position depending on the
```

File: plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/common/migration/DiagramMigrationTestCampaign02.java [hunk 2]
```diff
@@ -114,12 +114,12 @@ public void testEdgeLayout() throws Exception {
         openEditorOnDiagram("" + diagramID);
```
Comment (lredor) [inline plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/common/migration/DiagramMigrationTestCampaign02.java:114]: Copyright of the file must be shift to 2023

**Review by lredor — CHANGES_REQUESTED** [review]
In the commit comment, it will be better to be more explicit
`[tests] Fix DiagramMigrationTestCampaign0x tests`
instead of 
`[tests] Fix junit tests`

File: plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/common/migration/DiagramMigrationTestCampaign02.java (hunk 2 above)
Comment (lfasani) [reply plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/common/migration/DiagramMigrationTestCampaign02.java:114]: done

File: plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/common/migration/DiagramMigrationTestCampaign09.java (hunk 1 above)
Comment (lfasani) [reply plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/common/migration/DiagramMigrationTestCampaign09.java:118]: done

**Pull request merged**